traffic_portal/app/src/common/modules/form/role/form.role.tpl.html

```diff
@@ -25,7 +25,7 @@
         </ol>
         <div class="pull-right" role="group" ng-show="!settings.isNew">
             <button class="btn btn-primary" title="View Capabilities" ng-show="enforceCapabilities" ng-click="viewCapabilities()">View Capabilities</button>
-            <!--<button class="btn btn-primary" title="View Users" ng-click="viewUsers()">View Users</button>-->
+            <button class="btn btn-primary" title="View Users" ng-click="viewUsers()">View Users</button>
```

Comment:
This button (and actually the above 'viewCapabilities' button too) is a `button[type="submit"]`, should set `type` to `button`.